src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.ts

```diff
@@ -1,24 +1,41 @@
-import {Component, model, OnInit} from '@angular/core';
+import {Component, computed, input, model, OnInit} from '@angular/core';
+import {
+  NavigatorComponentGrouping,
+  NavigatorComponentSorting
+} from '../complex-navigator/complex-navigator-utils';
+import {ComplexSearchResult} from '../../shared/model/complex-results/complex-search.model';
+import {ComplexComponent} from '../../shared/model/complex-results/complex-component.model';
 
 @Component({
   selector: 'cp-complex-navigator-buttons',
   templateUrl: './complex-navigator-buttons.component.html',
   styleUrls: ['./complex-navigator-buttons.component.css']
 })
 export class ComplexNavigatorButtonsComponent implements OnInit {
-  interactorsSorting = model<string>();
+  componentsSorting = model<NavigatorComponentSorting>();
+  componentsGrouping = model<NavigatorComponentGrouping>();
   organismIconDisplay = model<boolean>();
   interactorTypeDisplay = model<boolean>();
   idDisplay = model<boolean>();
+  complexSearch = input<ComplexSearchResult>();
+
+  orthologButtonAvailable = computed(() => this.checkIfPantherGroups());
```

Comment:
Try this instead.

1. Delete method `checkIfPantherGroups`

2. Update `orthologButtonAvailable` to be
   ```
   orthologButtonAvailable = input<boolean>();
   ```

3. In `complex-navigator.component.ts` create a new boolean variable to store if there are ortholog groups
   ```
   orthologGroupsAvailable = true;
   ```

4. And pass it to the complex-navigator-buttons component in `complex-navigator-component.html`
   ```
    <cp-complex-navigator-buttons
      [(componentsSorting)]="componentsSorting"
      [(componentsGrouping)]="componentsGrouping"
      [(organismIconDisplay)]="organismIconDisplay"
      [(interactorTypeDisplay)]="interactorTypeDisplay"
      [(idDisplay)]="idDisplay"
      [orthologButtonAvailable]="orthologGroupsAvailable"
      [complexSearch]="complexSearch()">
    </cp-complex-navigator-buttons>
   ```

5. In `table-structure.component.ts` create a model to set the ortholog groups value
   ```
   orthologGroupsAvailable = model<boolean>();
   ```

6. And in that same component, set the value in the `createOrthologGroups` method if we create any group
   ```
    ...
    let anyOrthologGroupCreated = false; // new line
    const newNavigatorComponents: INavigatorComponent[] = [];
    for (const [_, proteins] of Object.entries(groupedByOrthology)) {
      if (proteins.length > 1) {
        // Multiple proteins in the ortholog group
        const group = proteins[0].orthologsGroup;
        const orthologsGroup = new NavigatorOrthologGroup(group, proteins);
        newNavigatorComponents.push(orthologsGroup);
        anyOrthologGroupCreated = true; // new line
      } else if (proteins.length === 1) {
        // Single proteins in the ortholog group, there's no need to create the group, we just use the protein component
        newNavigatorComponents.push(proteins[0]);
      }
    }
    this.orthologGroupsAvailable.set(anyOrthologGroupCreated); // new line
    newNavigatorComponents.push(...interactorsWithoutGroup);
    return newNavigatorComponents;
   ```

7. Finally, in `complex-navigator.component.html` pass the `orthologGroupsAvailable` variable to the model so it can be set
   ```
  <cp-table-structure [complexSearch]="complexSearch()"
                      [interactors]="interactors()"
                      [componentsSorting]="componentsSorting"
                      [componentsGrouping]="componentsGrouping"
                      [organismIconDisplay]="organismIconDisplay"
                      [interactorTypeDisplay]="interactorTypeDisplay"
                      [idDisplay]="idDisplay"
                      [canAddComplexesToBasket]="canAddComplexesToBasket()"
                      [canRemoveComplexesFromBasket]="canRemoveComplexesFromBasket()"
                      [(orthologGroupsAvailable)]="orthologGroupsAvailable"
                      (onComplexRemovedFromBasket)="onComplexRemovedFromBasket.emit($event)">
  </cp-table-structure>
   ```

I haven't tried this but it should work. `TableStructureComponent` sets if there are ortholog groups or not, it passes the value through the model to `ComplexNavigatorComponent` which passes it to `ComplexNavigatorButtonsComponent` to enable or disable the button.

It's like what we do for the sorting or grouping, but the other way around.